src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,87 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+
+    private Node<K, V>[] table;
+    private int size = 0;
+    private int threshold;
+
+    private static class Node<K, V> {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line before the declaration of the inner class `Node`.